src/app/cube/details/components/academic-level-card/academic-level-card.component.scss

```diff
@@ -0,0 +1,38 @@
+@import '~_vars.scss';
+
+.academic-level-card {
+    border-radius: 5px;
+    box-sizing: border-box; 
+    box-shadow: 0 4px 22px 0 rgba(0, 0, 0, 0.1);
+    height: 350px;
+    text-align: center;
+    width: 275px;
```

Comment:
Why fixed width here? Could we do a max-width so that it'll scale?

![image](https://user-images.githubusercontent.com/7431390/71187016-3cbb7b00-224c-11ea-82ca-6b0ebdbe46c2.png)